openvpn/hideme/Haarlem.ovpn

```diff
@@ -1,17 +1,100 @@
 client
-dev tun0
+dev tun
 proto udp
-remote nl.hide.me 3478
-cipher AES-128-CBC
+remote haarlem.hide.me 4000
+cipher AES-256-CBC
+auth SHA256
 resolv-retry infinite
 nobind
 persist-key
 persist-tun
 mute-replay-warnings
-ca /etc/openvpn/hideme/TrustedRoot.pem
 verb 3
-auth-user-pass /config/openvpn-credentials.txt
+auth-user-pass
```

Comment:
You need to leave this line as it was for the container to pass username/password to openvpn